CHANGELOG.md

```diff
@@ -0,0 +1,558 @@
+- (Fix tests for WritableStreamDefaultController/Writer (#1324), 2021-07-21)
+- (Remove mapping of console->Console (#1326), 2021-07-21)
+- (Fix variable names in custom tests to avoid "interface" (#1323), 2021-07-20)
+- (Bump @webref/idl from 2.2.8 to 2.2.9, 2021-07-22)
+- (Bump @webref/css from 2.0.4 to 2.0.5, 2021-07-22)
+- (Bump @mdn/browser-compat-data from 3.3.11 to 3.3.12, 2021-07-22)
+
+## v3.2.2
+
+- (Revert "Update lockfile version to 2" (#1321), 2021-07-20)
+- (Ignore user media tests on Firefox 34-52 (#1319), 2021-07-19)
+- (Update lockfile version to 2 (#1317), 2021-07-19)
+- (Cover inherited attributes (#1318), 2021-07-19)
+- (Add instrumentKey custom IDL (#1314), 2021-07-16)
+
+## v3.2.1
+
+- (Add tests for some JavaScript alt. names. in BCD (#1313), 2021-07-16)
+- (Remove Function.prototype.displayName test (#1311), 2021-07-16)
+
+## v3.2.0
+
+- (Remove tests for columnNumber/fileName/lineNumber (#1310), 2021-07-16)
+- (Generate test for JavaScript constructors (#1307), 2021-07-16)
+- (Generate tests for property symbols (@@ features) (#1306), 2021-07-16)
+- (Generate tests for JavaScript builtins (#1302), 2021-07-16)
+- (Bump @webref/css from 2.0.3 to 2.0.4 (#1303), 2021-07-15)
+- (Bump @mdn/browser-compat-data from 3.3.10 to 3.3.11 (#1305), 2021-07-15)
+
+## v3.1.11
+
+- (Add back custom IDL now gone from @webref/idl (#1300), 2021-07-15)
+- (Add MutationEvent instance (document.createEvent('MutationEvent')) (#1297), 2021-07-12)
+- (Add custom tests for WritableStreamDefaultController/Writer (#1295), 2021-07-09)
+- (Update UA parser to better handle iOS browsers (#1290), 2021-07-08)
+- (Enable fake media stream for Firefox in Selenium script (#1289), 2021-07-08)
+- (Custom tests: replace variable assignments as well (#1288), 2021-07-06)
+- (Bump @webref/idl from 2.2.7 to 2.2.8 (#1299), 2021-07-15)
+- (Bump @webref/idl from 2.2.6 to 2.2.7 (#1293), 2021-07-09)
+- (Bump @webref/css from 2.0.2 to 2.0.3 (#1291), 2021-07-09)
+
+## v3.1.10
+
+- (Add IDs for export buttons (#1282), 2021-07-05)
+- (Fix Firefox collection regarding new WebKitAnimationEvent test (#1280), 2021-07-04)
+- (Add custom test for WebGLRenderingContext (#1279), 2021-07-04)
+- (Increase the verbosity of errors in results parsing (#1278), 2021-07-04)
+- (More Selenium script updates (#1277), 2021-07-04)
+- (Selenium script updates (#1276), 2021-07-04)
+
+## v3.1.9
+
+- (Remove custom IDL now in @webref/idl, 2021-07-01)
+- (Bump @webref/idl from 2.2.4 to 2.2.6, 2021-07-01)
+- (Bump @webref/css from 2.0.1 to 2.0.2, 2021-07-01)
+- (Bump @mdn/browser-compat-data from 3.3.9 to 3.3.10, 2021-07-01)
+- (Bump @webref/idl from 2.2.3 to 2.2.4, 2021-06-25)
+- (Bump @webref/css from 2.0.0 to 2.0.1, 2021-06-24)
+- (Bump @mdn/browser-compat-data from 3.3.8 to 3.3.9, 2021-06-24)
+
+## v3.1.8
+
+- (Add a --path argument to filter BCD paths by wildcards (#1245), 2021-06-21)
+- (Use <b> as a HTMLElement instance (not HTMLUnknownElement) (#1253), 2021-06-21)
```

Comment:
The `<b>` here makes the rest of the changelog bold. Is there a Markdown escaping function we can use?